lib/recurly/resource.rb

```diff
@@ -1118,5 +1118,10 @@ def fetch_associated(name, value, options = {})
     def xml_keys
       changed_attributes.keys.sort
     end
+
+    def add_money_tag?(key, value)
+      value && key.end_with?('_in_cents') && !respond_to?(:currency) && !value.is_a?(Money) &&
+        (!is_a?(PercentageTier) && !is_a?(SubAddOnPercentageTier))
```

Comment:
**Minor nit** I tend to prefer splitting all of the different segments onto their own lines.  Just marginally more readable IMO.  Very optional.

```suggestion
      value && 
        key.end_with?('_in_cents') && 
        !respond_to?(:currency) && 
        !value.is_a?(Money) &&
        !is_a?(PercentageTier) && 
        !is_a?(SubAddOnPercentageTier)
```